infrastructure/base/fluentbit/fluentbit-config.yaml

```diff
@@ -4,6 +4,28 @@ envFrom:
 
 logLevel: info
 
+## Fluent Bit configuration
+service:
```

Comment:
I want to make sure we are not opening another service to the outside world here. The point of this is just to allow the prometheus running in the cluster to scrape these new values, right?

Reply:
Yes, its just configuring the existing cluster IP service in the fluentbit namespace by overwriting the default helm chart values.